meta-resin-common/recipes-support/resin-conf-reset/resin-conf-reset/resin-conf-reset

```diff
@@ -9,8 +9,10 @@ populate() {
     # Re-construct directory structure and populate files
     if [ -f $2 ]; then
         mkdir -p "$(dirname $1)" && cp $2 $1
-    else
+    elif [ -d $2 ]; then
         mkdir -p $1 && find $2 \( -type f -o -type l \) -exec cp {} $1 +
+    else
+        mkdir -p $1
```

Comment:
I agree with your points, but I think maybe we should create a different task and refactor the script under another PR.